TwilioVerifyDemo/TwilioVerifyDemo/CreateFactor/Presenter/CreateFactorPresenter.swift

```diff
@@ -43,6 +43,30 @@ class CreateFactorPresenter {
     self.twilioVerify = twilioVerify
     self.accessTokensAPI = accessTokensAPI
   }
+  
+  /// This method is intended to demonstrate how to access to
+  /// Verify API errors, such custom response codes and details.
+  /// See https://www.twilio.com/docs/api/errors
+  /// - Parameter error: Create Factor result error
+  private func retrieveAPIError(
+    with error: TwilioVerifyError
+  ) -> String? {
+    switch error.originalError as? NetworkError {
+      case .failureStatusCode(let response):
+        // Gets server response code
+        var errorResponse = response
+        
+        if let code = errorResponse.apiError?.code,
+           let message = errorResponse.apiError?.message {
+          return "Code: \(code) - \(message)"
+        }
+        
+        return nil
+        
+      default:
+        return nil
+    }
```

Comment:
Suggestion: This could be improved using `if case let` instead of a full `switch` statement.

Reply:
Thanks for suggestion, I just updated it